wallets/provider-argentx/src/signer.ts

```diff
@@ -1,12 +1,12 @@
 import type { SignerFactory } from 'rango-types';
 
+import { DefaultStarknetSigner } from '@rango-dev/signer-starknet';
```

Comment:
`DefaultStarknetSigner` has no dependency and result in small chunks so removed dynamic import.